docs/pages/jpscore/jpscore_tests_rimea.md

```diff
@@ -0,0 +1,125 @@
+---
+title: RiMEA
+keywords: simulation  validation
```

Comment:
Is it possible to integrate the results of the rimea tests here automatically?

Reply:
would be a good idea to put a link to the actual pdf-file that results from the script 
[run-report](https://github.com/JuPedSim/jpscore/blob/master/systemtest/rimea_tests/run%2Breport_rimea.py).
Although this pdf should be produced on a regular basis by the CI, I think we can already produce one, upload it to sciebo, and link it here.

Reply:
Can we upload it to github?

Reply:
Sure thing!  #632